docs/superpowers/plans/2026-05-21-gap7-iface-reachability-plan.md

```
@@ -0,0 +1,771 @@
+**Files:**
+- Modify: `examples/build_product.dip` (insert ~6 lines + heredoc after existing `PROBE_EOF` block at line 93, before `printf 'setup-ready'` at line 95)
+
+- [ ] **Step 1: Read the current Setup node bracketing the insertion point**
+
+Use Read tool on `examples/build_product.dip` lines 80-100 to confirm exact whitespace and indentation around the insertion point.
+
+- [ ] **Step 2: Insert the new heredoc block**
+
+Use Edit tool. Find this text (which currently appears at lines 93-95):
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)